### PR TITLE
CC6002: Update inheritance.md #21

### PR DESCRIPTION
**Pull request by carolinacmoravia**
@rpetrusha, 
Could you review this update, please? You can see the reference of the contibution here: https://github.com/dotnet/docs.pt-br/pull/21/files.
Many thanks.

# Title

On the title describe
what you've fixed (or created) with this Pull Request (PR).

## Summary

Insert a short (one or two sentence) summary here.

Fixes #Issue_Number

>Note: The "Fixes #nnn" syntax in the PR description causes
>GitHub to automatically close the issue when this PR is merged.
> Remove that line if you don't have issues associated with this
> PR. Click on the Guidelines for Contributing link above for details.

## Details

Explain your changes, and why you made them. If that
information is already available in the issue referenced
above, just referencing the issue is preferred to copying
the text.

This may not be necessary depending on the scope of the PR 
changes. For example, "fix typo in introduction.md" is
sufficient to describe that PR.

## Suggested Reviewers

If you know who should review this, use '@' to request a review.

### CODE REVIEW

File: docs/csharp/tutorials/inheritance.md [hunk 1]
```diff
@@ -220,7 +220,7 @@ The following example shows the source code for the `Publication` class, as well
 
 - Publishing-related members
 
-  Two methods, `Publish` and `GetPublicationDate`, set and return the publication date. The `Publish` method sets a private  `published` flag to `true` when it is called and assigns the date passed to it as an argument to the private `datePublished` field. The `GetPublicationDate` method returns the string "NYP" if the `published` flag is `false`, and the value of the `datePublished` field if it is `true`.
```
Comment (svick) [inline docs/csharp/tutorials/inheritance.md]: This is wrong. The change corresponding to https://github.com/dotnet/docs.pt-br/pull/21 would be to change line 213 above.

And that change would still be wrong, `Title` is a get-only auto-property, it doesn't have a named backing field. So I think the whole "which stores the value in a private field named `pubTitle`" part should be removed.

**Review by svick — CHANGES_REQUESTED** [review]

File: docs/csharp/tutorials/inheritance.md (hunk 1 above)
Comment (carolinacmoravia) [inline docs/csharp/tutorials/inheritance.md]: Many thanks for your prompt reply. Contribution rejected. 
Closing this PR.

**Comment by mairaw** [comment]
I'm reopening this one @carolinacmoravia. I've made some similar changes to this topic yesterday. The underlying code has changed and now the text needs to reflect the use of auto-properties.

**Comment by svick** [comment]
@mairaw Because this PR was closed, I have created https://github.com/dotnet/docs/pull/2933 to remove mentions of the removed fields. I think it make sense to keep only one of those two PRs open. Should this one be closed again, or should I close https://github.com/dotnet/docs/pull/2933?

**Pull request closed without merging**